src/bb-load.php

```diff
@@ -57,11 +57,11 @@ function handler_exception($e)
     ";
     $page = str_replace(PHP_EOL, "", $page);
     print $page;
-    if($e->getCode()) {
+    if($e->$code) {
         print sprintf('<p>Code: <em>%s</em></p>', $e->getCode());
     }
-    print sprintf('<p>%s</p>', $e->getMessage());
-    print sprintf('<p><a href="http://docs.boxbilling.com/en/latest/search.html?q=%s&check_keywords=yes&area=default" target="_blank">Look for detailed error explanation</a></p>', urlencode($e->getMessage()));
+    print sprintf('<p>%s</p>', $e->$message);
```

Comment:
`$e->$message` should be  `$e->getMessage()`
`$e->$code` should be $e->getCode()

Reference: https://www.php.net/manual/en/class.exception.php